### ISSUE TITLE
No GPG keys. You need to create at least one.

### ISSUE THREAD

**Issue by mayneyao** [opersys/gitgeist-poc#2]
I have encountered this problem. the cause of this problem is that my gpg's version is 2, I need to use `gpg2` to call it in the command line.

```
which gpg2
/usr/local/bin/gpg2
mv /usr/local/bin/gpg2 /usr/local/bin/gpg
```
Hope it helps😃

**Comment by fdgonthier** [comment]
I've just patched the program so that it tries to use 'gpg' first but then try to execute 'gpg2' when that fails. I really want people to have as little troubles as possible to try this proof-of-concept. 

In the future, I'd ideally not have to execute 'gpg' directly. Because I wanted to keep dependencies to a minimum for the proof-of-concept, we've not used any Node.js modules. If we implement a full version of this concept, I'll be using modules that can handle this.

I'd be happy if you tested the patch I just made.
